advocacy_docs/repos/getting_started/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Getting started with repositories
+navTitle: Getting Started
+description: How to download and install the EDB package you need
+navigation:
+- with_web
+- with_cli
+- troubleshooting
+---
+
+* [Getting started...using the Web Repos page](with_web): If you have an Enterprise subscription to EDB software, and want to use the web interface to guide you in downloading and installing packages, use this guide.
+
+* [Getting started...using the CLI](with_cli): For all subscriptions, you can use the CLI to install packages from EDB repositories. This guide works for all EDB repositories and is the most flexible way to install packages.
```

Comment:
I think we should spell out that you need to use this method if you have Standard, Community360, or if you are looking to install Postgres Distributed packages (without TPA). I know this is horribly complex, but that's just what we have and we need to shepherd customers through it.